tests/scripts/github_cc_reviewers.py

```diff
@@ -106,5 +107,8 @@ def find_reviewers(body: str) -> List[str]:
         for reviewer in to_add:
             try:
                 github.post(f"pulls/{number}/requested_reviewers", {"reviewers": [reviewer]})
-            except error.HTTPError as e:
+            except KeyboardInterrupt:
+                sys.exit()
+            except Exception as e:  # pylint: disable=broad-except
```

Comment:
nit to make pylint happy
```suggestion
            except (RuntimeError, error.HTTPError) as e:
```